tests/test_simple_csv_workflow.py

```diff
@@ -0,0 +1,44 @@
+from unittest.mock import patch
+
+
+def test_simple_csv_workflow_item_metadata_iter_success(
+    simple_csv_workflow_instance, mocked_s3_simple_csv, item_metadata
+):
+    metadata_iter = simple_csv_workflow_instance.item_metadata_iter(
+        metadata_file="metadata.csv"
+    )
+    assert next(metadata_iter) == item_metadata
+
+
+def test_simple_csv_workflow_get_item_identifier_success(
+    simple_csv_workflow_instance, item_metadata
+):
+    assert simple_csv_workflow_instance.get_item_identifier(item_metadata)
```

Comment:
Could we `assert` the actual value retrieved by `get_item_identifier`?

Reply:
Change has been made!